packages/domain-search/src/hooks/use-policy-notices.tsx

```diff
@@ -0,0 +1,43 @@
+import { PolicyNotice } from '@automattic/api-core';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { HTTPS_SSL } from '@automattic/urls';
+import { useTranslate } from 'i18n-calypso';
+import { useSuggestion } from './use-suggestion';
+
+export const usePolicyNotices = ( domainName: string ): PolicyNotice[] => {
+	const translate = useTranslate();
+	const suggestion = useSuggestion( domainName );
+	const policyNotices = suggestion.policy_notices || [];
+
+	const getPolicyNoticeMessage = ( { type, message }: PolicyNotice ) => {
+		if ( type === 'hsts' ) {
+			return translate(
```

Comment:
Let's use `__` from `@wordpress/i18n` instead since it's the Gutenberg version.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/105532/commits/7160721c9db3a749e2d7d51ba763c220574c11a9.

To be honest, I'm a bit confused about when to use what for localization. Is using `translate` a deprecated practice? Also, when should I use `const { __ } = useI18n()` vs `import { __ } from '@wordpress/i18n'`?